cristin-organization/src/test/java/no/unit/nva/cristin/organization/organization.feature

```diff
@@ -59,3 +60,19 @@ Feature: API tests for Cristin Organization retrieve and search
     And match response.hits == '#array'
     And match response.size == '#number'
     And match response.hits == '#[2]' // hits array length == 0
+
+  Scenario: GET organization for known organization returns list of search results with depth
+    Given  path '/organization'
+    And param query = existingOrganizationIdentifier
+    When method GET
+    Then status 200
+    And match response.hits == '#array'
+
+  Scenario: GET organization for known organization returns list of search results without depth
+    Given  path '/organization'
+    And param query = existingOrganizationIdentifier
+    And param depth = 'top'
+    When method GET
+    Then status 200
+    And match response.hits == '#array'
+    And match response.hits[0].partOf != '#present'
```

Comment:
So we dont want upper organizations this hit if part of? Is not that what top is? get all organizations this is part of? And rather exclude hasPart?

Reply:
Or is parameter top to get top level organization for given identifier?